meta/models.py

```diff
@@ -21,6 +21,9 @@ class ModelMeta(object):
     _metadata = {}
     _metadata_default = {
         'title': False,
+        'og_title': False,
+        'twitter_title': False,
+        'gplus_title': False,
```

Comment:
Could you also add these to ``Meta`` and ``MetadataMixin``?
See https://github.com/nephila/django-meta/blob/develop/meta/views.py#L19 and https://github.com/nephila/django-meta/blob/develop/meta/views.py#L129
We need to keep the interfaces in sync

Reply:
I have added additional fields for Meta and MetadataMixin. Field description (og_description ...) does not require the same change?